src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/ScreenRecorder.cs

```diff
@@ -270,6 +280,7 @@ private async Task HandleEndRecording(long tickCount,
             _needToRefreshAssets = true;
 #endif
 
+            //TODO (REG-2181): Add upload logic for metadata like action analysis, code coverage, etc - we probably also want consider if we redo the game metadata upload here and roll these into a single zip file of metadata ?
```

Comment:
Yeah I wonder if it might get messy with the number of zip files we are uploading now... something to think about.